test/runtests.sh

```diff
@@ -13,10 +13,10 @@ build_type=$1
 if [[ $build_type != "-d" && $build_type != "-t" ]]; then
     echo "Warning: You haven't provide either '-d' (debug) or '-t' (test)."
     echo "Warning: Searching for ch.."
-    if [[ -f "$test_path/../BuildLinux/Debug/ch" ]]; then
+    if [[ -f "$test_path/../Build/clang_build/x64_debug/bin/ch" ]]; then
```

Comment:
>x64_debug [](start = 46, length = 9)

Prefer to allow passing a parameter to this script (so it can be used to run locally with altered parameters). Set the defaults for these parameters to `x64` and `debug`